rules/cross-platform/command_and_control_potential_gc2_sheet_usage.toml

```diff
@@ -0,0 +1,106 @@
+[metadata]
+creation_date = "2023/04/23"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/04/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects the potential use of the Google Command and Control (GC2) tool. GC2 is a public open-source red teaming tool
+that uses Google Workspace services for C2. While the tool is only intended for red teaming, it has been leveraged by
+advanced persistent threats (APT) for intrusions.
+"""
+false_positives = ["    "]
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Google Command and Control (GC2) Sheet Activity"
+references = ["https://github.com/looCiprian/GC2-sheet"]
+risk_score = 73
+rule_id = "d54a104e-e223-11ed-8d6e-f661ea17fbcd"
+severity = "high"
+tags = [
+    "Elastic",
+    "Host",
+    "Linux",
+    "Threat Detection",
+    "Command and Control",
+    "Exfiltration",
+    "Execution",
+    "Windows",
+    "Linux",
+    "macOS",
+]
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan=1m
```

Comment:
could you consider fine tuning the "sequence by" even more with the addition of a user field? to eliminate accidentally capturing some of these events happening in isolation across different users on the same host. Or do you think host.id is enough in this particular case?

Reply:
Great idea, I think `host.id` is enough in this case since this is a malicious endpoint binary.